### ISSUE TITLE
Multicolor Tile Marking

### ISSUE THREAD

**Issue by ElevatedLife** [runelite/runelite#5812]
I have found a lot of value in the ground marker feature, it is a great tool to help you remember where to stand. 

I feel this tool could be improved by adding the functionality to have markers with different colors. Currently if you mark a tile green. and then change the color to red. all your existing markers will now be red.

I am proposing that when you change the color only tiles marked after the color change will be affected by the new color. 

This would be especially useful for different solo methods that require you to change prayer or attack on a certain tile. i.e yellow marker i attack, blue marker i pray mage, red marker i pray melee.

tldr; being able to have different colored ground markers at the same time.

**Comment by deathbeam** [comment]
We already have issue open for this, closing as duplicate
